ledger/src/posw/mod.rs

```diff
@@ -94,7 +103,7 @@ mod tests {
             .mine(&subroots, difficulty_target, &mut rand::thread_rng(), std::u32::MAX)
             .unwrap();
 
-        assert_eq!(proof.len(), 972); // NOTE: Marlin proofs use compressed serialization
+        assert_eq!(proof.len(), 771); // NOTE: Marlin proofs use compressed serialization
```

Comment:
the `771` should ideally be replaced with an imported `const`; also, is this assertion safe to make?